metadata.yaml

```diff
@@ -7,7 +7,7 @@ assumes:
 display-name: Jenkins K8s
 summary: Jenkins Continuous Integration Server
 maintainers:
-  - IS DevOps <launchpad.net/~canonical-is-devops>
+  - https://launchpad.net/~canonical-is-devops
```

Comment:
Next time, I suggest you open a separate PR for this kind of changes, so that PRs are move atomic and can be easily reverted if needed

Reply:
Yup, I should have opened a new branch! Thanks for the reminder!